apps/mull-ui/src/app/components/event-card/event-card.spec.tsx

```diff
@@ -1,22 +1,46 @@
+import { MockedProvider } from '@apollo/client/testing';
 import { render } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
 import React from 'react';
+import { Router } from 'react-router-dom';
 import renderer from 'react-test-renderer';
 import { dummyEvent } from '../../../constants';
 import EventCard from './event-card';
 
 describe('EventCard', () => {
+  const history = createMemoryHistory();
   it('should render successfully', () => {
-    const { baseElement } = render(<EventCard event={dummyEvent} />);
+    const { baseElement } = render(
+      <MockedProvider>
+        <Router history={history}>
+          <EventCard event={dummyEvent} />
+        </Router>
+      </MockedProvider>
```

Comment:
You could this to a separate function if you want, see `renderHelper` in `login.spec.tsx`. The history can be a parameter, or in your case, it could be created inside the helper function

Reply:
Addressed